tests/pytests/integration/cli/test_salt_call.py

```diff
@@ -61,16 +59,28 @@ def test_json_out_indent(salt_call_cli, indent):
     assert ret.stdout == expected_output
 
 
-def test_local_sls_call(salt_call_cli):
-    fileroot = os.path.join(RUNTIME_VARS.FILES, "file", "base")
-    ret = salt_call_cli.run(
-        "--local", "--file-root", fileroot, "state.sls", "saltcalllocal"
-    )
-    assert ret.exitcode == 0
-    state_run_dict = next(iter(ret.json.values()))
-    assert state_run_dict["name"] == "test.echo"
-    assert state_run_dict["result"] is True
-    assert state_run_dict["changes"]["ret"] == "hello"
+def test_local_sls_call(salt_call_cli, base_env_state_tree_root_dir):
+    sls_contents = """
+    regular-module:
+      module.run:
+        - name: test.echo
+        - text: hello
+    """
+    with pytest.helpers.temp_file(
+        "saltcalllocal.sls", sls_contents, base_env_state_tree_root_dir
+    ):
+        ret = salt_call_cli.run(
+            "--local",
+            "--file-root",
```

Comment:
Did you have to pass `--file-root` or is that part of the test?

Reply:
Part of the test I believe.